spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/convert/ReadWriteConversions.java

```diff
@@ -65,6 +65,13 @@
 	 */
 	Value convertOnWrite(Object obj, DatastorePersistentProperty persistentProperty);
 
+	/**
+	 * Converts an object to a Cloud Datastore {@link Value}, for non-collection objects
```

Comment:
In the method above, you should probably then say that it supports collection objects as well.
I'm not clear on why we need both methods exposed though. Why can't the method above be used to convert either collection or single?

Reply:
We need this one because we don't have a persistent property for the map.